resty

```diff
@@ -168,6 +170,17 @@ for my $var (sort keys %ENV) {
     $env_list .= "env $var;\n";
 }
 
+my $main_include_directive = '';
+my $http_include_directive = '';
+
+if ($main_include) {
+    $main_include_directive = "include $main_include;"
```

Comment:
@nilnor Thanks for the reminder of wildcards. I'm just worried about use of relative paths in these option values. Maybe we can use Perl's `glob` builtin function to expand the wildcards ourselves and generate a bunch of `include` directives for each individual file? What do you think?